packages/main/lib/create-new-component/create-new-component.js

```diff
@@ -0,0 +1,65 @@
+const jsFileContentTemplate = sComponentName => {
+	return `import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import ${sComponentName}Template from "./generated/templates/${sComponentName}Template.lit.js";
+
+// Styles
+import ${sComponentName}Css from "./generated/themes/${sComponentName}.css.js";
+
+
+const metadata = {
+	tag: "ui5-${sComponentName.toLowerCase()}",
+	properties: {
+		//
+	},
+	slots: {
+		//
+	},
+	events: {
+		//
+	},
+};
+
+class ${sComponentName} extends UI5Element {
+	static get metadata() {
+		return metadata;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get styles() {
+		return ${sComponentName}Css;
+	}
+
+	static get template() {
+		return ${sComponentName}Template;
+	}
+
+	static async define(...params) {
+		super.define(...params);
+	}
+}
+
+${sComponentName}.define();
+
+export default ${sComponentName};
+`;
+};
+
+const fs = require("fs");
+
+
+const aArgs = process.argv.slice(2);
+const sComponentName = aArgs[0];
```

Comment:
We don't enforce hungarian notation in this project, but I'm fine to leave it as is since this is a build script. Up to you :)